bsp/worker/src/mill/bsp/worker/SyntheticRootBspBuildTargetData.scala

```diff
@@ -15,11 +14,11 @@ import ch.epfl.scala.bsp4j.BuildTarget
  */
 class SyntheticRootBspBuildTargetData(topLevelProjectRoot: os.Path) {
   val id: BuildTargetIdentifier = new BuildTargetIdentifier(
-    Utils.sanitizeUri(topLevelProjectRoot / s"synth-build-target-${UUID.randomUUID()}")
+    Utils.sanitizeUri(topLevelProjectRoot / "mill-synthetic-root-target")
```

Comment:
We're removing the UUID here. Do we know that it is not necessary?

Reply:
My guess is that this avoid conflicts with an actual module users created. But a collision seems quite unlikely.